### ISSUE TITLE
Question about Multi-Head Abalations

### ISSUE THREAD

**Issue by Hprairie** [state-spaces/mamba#619]
I'm running some experiments on the scaling laws of Mamba2 and was unable to find any hyperparameter values for the following ablation on multi-head structure.

![image](https://github.com/user-attachments/assets/2d694170-08e3-4e4e-8470-ef98127e43e5)

Any details on learning rate, batch size, training steps, weight decay, gradient clipping, lr schedule, and optimizer values would be awesome!

Also what expansion factor is used and what state-dimension is used for these ablations? Thanks so much for the help!
